src/quickView/quickView.controller.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Query,
+  Body,
+  HttpCode,
+  HttpStatus,
+} from "@nestjs/common";
+import { QuickViewService } from "./quickView.service";
+import * as path from "path";
+// import { JwtAuthGuard } from "../auth/guards/jwt-auth.guards";
+// import { UseGuards } from "@nestjs/common/decorators";
+
+@Controller()
+export class QuickViewController {
+  constructor(private quickViewService: QuickViewService) {}
+
+  // @UseGuards(JwtAuthGuard)
+  @Post("quickView")
+  @HttpCode(HttpStatus.OK)
+  async getPostInfo(
+    @Query("filename") filename: string,
+    @Body("accessToken") token: { accessToken: string }
+  ): Promise<object> {
+    // const login_user_email = req.user["email"];
+
+    const login_user_email = await this.quickViewService.getLoginUserEmail(
```

Comment:
Too match async await. Create a new function of service and use filename to get all the information like user_name、user_id etc.

Reply:
Because too much async await will delay function execution. Try to get all the data Use Promise.all or get all the data in db operation at one time(Just suggestion).